pkg/eventing/v1alpha1/client.go

```diff
@@ -198,6 +205,13 @@ func (b *TriggerBuilder) RemoveFilter(key string) *TriggerBuilder {
 	return b
 }
 
+func (b *TriggerBuilder) RemoveMultipleFilters(keys []string) *TriggerBuilder {
```

Comment:
Do wee need this ? (and also RemoveFilter). At the end its a builder with a very short lifetime. If you don't want to have something added, just don't add it. IMO, we should only have methods which 'build up' something.

Reply:
The remove filter will remove existing filters setting by `--filter name-`.
If I need to remove some existing filters, what should I do ? Get the existing map of filters, then remove some keys outside builder, and then set back to builder ?

Reply:
I keep only RemoveFilters in the latest version to remove those filters setting by `--filter name-`